vmware/ngc/NGC-Register/src/main/resources/application.properties

```diff
@@ -0,0 +1,18 @@
+server.port=8088
+#https
+server.ssl.key-store=classpath:keystore.jks
+server.ssl.key-alias=server
+server.ssl.enabled=true
+server.ssl.key-store-password=123456
```

Comment:
we add as plain text?!

Reply:
In the next step we have plan to encrypt this plain text.

Reply:
Raised #255